### ISSUE TITLE
Tagged PDF for manuscript submission - need end of day Thursday

### ISSUE THREAD

**Issue by emily-phet** [phetsims/tasks#923]
We need a pdf file with tags and alt text submitted for a publication. We have run into difficulties doing this (@terracoda, @jessegreenberg, and myself) and may need a team effort for a short time on Thursday. 
Here's the difficulty - a single page (page 7) of the eight page document will not allow us select or tag any content. Essentially, the page seems to be like one big unselectable image to Adobe Acrobat. 

We would like to try:
1) exporting the word file on different devices. Maybe magically someone has better luck with this particular challenge we've encountered. It does seem different versions of word, etc. result in different outcomes for accessible pdf's overall. If that doesn't work:
2) work together to iteratively export the content paragraph by paragraph and isolate the part of the doc that is causing the problem. 

I think if we have a few people together working on this, it won't take more than 20 minutes or so.

**Comment by ariel-phet** [comment]
@emily-phet have you contacted anyone at disability services? Someone like Alaina? This seems more in their wheelhouse

**Comment by ariel-phet** [comment]
Will ask @arnabp as well...since he tends to be our resident random tech savvy person

**Comment by terracoda** [comment]
@ariel-phet and @emily-phet, ideally, we need a person who has access to a Windows computer with a recent or latest version of MS Word and Adobe Acrobat Professional, or Adobe Acrobat DC.

I have put all the files on google drive conference folder.

**Comment by emily-phet** [comment]
Responding to the comment about asking folks in disability services - 
I haven't reached out to them, since the problem isn't a typical issues. It's not a "how do I make this accessible" kind of problem - it's a "how do I deal with some issues likely in the ACM manuscript template" kind of problem.

**Comment by arnabp** [comment]
I have an up to date windows computer, but I don't have Adobe Acrobat Pro or DC so I may not be much help. Which files should I be looking for in the google drive conference folder?

**Comment by terracoda** [comment]
Thanks @arnabp, @emily-phet and I found the main source of the problem after much effort.
One figure on page 7 was corrupting the page some how. We replaced the image and I was able to tag the pdf.

Adobe's a11y checker says there are still a coupe of issues. 

I have sent everything to @emily-phet.

@emily-phet, please close issue when you submit the manuscript.

**Comment by terracoda** [comment]
@emily-phet submitted manuscript, so closing this issue.
